.gitignore

```diff
@@ -6,3 +6,5 @@ node_modules
 *.DS_Store
 logs
 *.log
+deviceconfig.js
+*.vscode
```

Comment:
Cool. Fixed it.

---
In reply to: [73765465](https://github.com/openT2T/translators/pull/37#discussion_r73765465) [](ancestors = 73765465,73765456)